src/Test/ODMSchemaResetter.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Zenstruck\Foundry\Test;
+
+use Doctrine\Persistence\ManagerRegistry;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+
+/**
+ * @internal
+ */
+final class ODMSchemaResetter implements SchemaResetterInterface
+{
+    use CommandRunner;
+
+    /** @var Application */
+    private $application;
+    /** @var ManagerRegistry */
+    private $registry;
+
+    public function __construct(Application $application, ManagerRegistry $registry)
+    {
+        $this->application = $application;
+        $this->registry = $registry;
+    }
+
+    public function resetSchema(): void
+    {
+        foreach ($this->objectManagersToReset() as $manager) {
+            $this->runCommand(
+                $this->application,
+                'doctrine:mongodb:schema:drop',
+                [
+                    '--dm' => $manager,
+                ]
+            );
+
+            $this->runCommand(
+                $this->application,
+                'doctrine:mongodb:schema:create',
+                [
+                    '--dm' => $manager,
+                ]
+            );
+        }
+    }
+
+    /** @return list<string> */
+    private function objectManagersToReset(): array
+    {
+        return [$this->registry->getDefaultManagerName()];
```

Comment:
for now, we only deal with `default` manager, but we can imagine to introduce new `FOUNDRY_RESET_ODM_MANAGERS` env var